engine/render/src/render/render.cpp

```diff
@@ -420,9 +439,9 @@ namespace dmRender
 
         RenderListRange* ranges = context->m_RenderListRanges.Begin();
         uint32_t num_ranges = context->m_RenderListRanges.Size();
-        for( uint32_t i = 0; i < num_ranges; ++i)
+        for( uint32_t r = 0; r < num_ranges; ++r)
```

Comment:
An old bug, where the variable name was shadowing the next loop (also using `i`)